google/cloud/spanner/samples/samples.cc

```diff
@@ -396,6 +395,42 @@ void QueryDataWithStruct(google::cloud::spanner::Client client) {
 }
 //! [END spanner_query_data_with_struct]
 
+//! [START spanner_query_data_with_array_of_struct]
+void QueryDataWithArrayOfStruct(google::cloud::spanner::Client client) {
+  namespace spanner = google::cloud::spanner;
+  // [START spanner_create_array_of_struct_with_data]
+  // Cloud Spanner STRUCT<> types with named fields are represented by
+  // std::tuple<std::pair<std::string, Ts>...>, create an alias to make it
```

Comment:
Small point, but I might use `T` instead of `Ts` as the `.second` type in the pair.

Reply:
Done.